web/themes/custom/news/src/styles/entry.scss

```diff
@@ -470,30 +477,56 @@
           }
         }
       }
+
+      .row-5 {
+        display: none;
+      }
     }
   }
-  .more-link {
+
+  .pager {
+    margin-top: 10px;
```

Comment:
@TerryKolodii do you have a spacing() function in SCSS already?

Reply:
Now i have